src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -25,6 +25,42 @@ class UserInfoSection extends React.Component {
       );
     }
 
+    let entitlementInfo;
+    if (enrollmentData.percentageBenefit === 0 && enrollmentData.originalEntitlement === 0) {
+      entitlementInfo = (
+        <div>
+          <h4>When You Can Receive Benefits</h4>
+          <div className="usa-alert usa-alert-warning usa-content">
+            <div className="usa-alert-body">
+              <h2>Currently Disallowed</h2>
+              We have received your application and have determined that you are not currently eligible
+              for Post-9/11 GI Bill benefits. Additional service time could change this determination.
+            </div>
+          </div>
+        </div>
+      );
+    } else {
+      entitlementInfo = (
+        <div>
+          <div>
+            <h4>When You Can Receive Benefits</h4>
```

Comment:
I started doing it that way, and then had doubts because I thought it would be confusing to be returning 2 separate variables when only one of them will always be null, but I agree the naming of this makes it weird. I'll change it, thanks!